src/solver/optimisation/opt_export_structure.cpp

```diff
@@ -93,30 +93,18 @@ void OPT_Export_add_variable(std::vector<std::string>& varname,
                              int Var,
                              Antares::Data::Enum::ExportStructDict structDict,
                              int firstVal,
-                             int secondVal)
-{
-    if ((int)varname.size() > Var && varname[Var].empty())
-    {
-        std::stringstream buffer;
-        buffer << Var << " ";
-        buffer << Antares::Data::Enum::toString(structDict) << " ";
-        buffer << firstVal << " ";
-        buffer << secondVal;
-        varname[Var] = buffer.str();
-    }
-}
-
-void OPT_Export_add_variable(std::vector<std::string>& varname,
-                             int Var,
-                             Antares::Data::Enum::ExportStructDict structDict,
-                             int firstVal)
+                             std::optional<int> secondVal)
 {
     if ((int)varname.size() > Var && varname[Var].empty())
     {
         std::stringstream buffer;
         buffer << Var << " ";
         buffer << Antares::Data::Enum::toString(structDict) << " ";
         buffer << firstVal;
+        if (secondVal)
```

Comment:
I like the **has_value()*... **value()** version

Reply:
It's a bit more expressive, I agree